int32 id
std_msgs/Header header
LShape l_shape
Pose2D pose